### DISCUSSION TITLE
Combining on: push with on: schedule

### DISCUSSION THREAD

**Discussion by Tyrrrz** [community/community#25806] [Actions]
<p>Is it possible to have a workflow that runs both on push and on schedule?</p>
<p>Basically I have a workflow that I want to run on each commit but also nightly to make sure integration tests still run successfully. I have some volatile dependencies so stuff can break overnight.</p>
<p>Currently I have two copy-pasted workflow but with different “on” value. Is there any way to combine them?</p>

**Comment by one-three-three-seven** [comment, marked as answer]
<p>You can solve it with</p>
<pre><code>on:
  push:
  schedule:
    - cron: '* * * * *'
</code></pre>

**Comment by weide-zhou** [comment]
<p>Just reminder, the shortest interval you can run scheduled workflows is once every 5 minutes.</p>

**Comment by Tyrrrz** [comment]
<p>I was so close! Thank you</p>
